domain/src/test/AndroidManifest.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  package="org.oppia.domain">
+  <uses-sdk android:minSdkVersion="19"
+    android:targetSdkVersion="28" />
```

Comment:
In some PR's I see target SDK as 29? 
Should we update here too?

Reply:
For some reason, using Api 29 breaks Bazel tests... See the thread I tagged you in [here ](https://github.com/oppia/oppia-android/pull/1495)